lib/solr/response/solr_error.rb

```diff
@@ -1,23 +1,32 @@
 module Solr
   class Response
     class SolrError
+      class None < SolrError
+        def initialize
+          super(code: nil, message: nil)
+          freeze
+        end
+
+        def inspect
+          'SolrError: None'
+        end
+      end
+
+      def self.none()
```

Comment:
Usually I don't add `()` to methods without params. Just a code style choice, it's up to you.

Reply:
fixed, I don't know how I missed that